okhttp/src/main/java/okhttp3/internal/connection/Transmitter.kt

```diff
@@ -122,7 +122,7 @@ class Transmitter(
       if (this.request!!.url.canReuseConnectionFor(request.url) && exchangeFinder!!.hasRouteToTry()) {
         return // Already ready.
       }
-      check(exchange == null)
+      check(exchange == null) { "Transmitted has existing exchange, has an app interceptor left prior responses open?" }
```

Comment:
```suggestion
      check(exchange == null) { "Transmitter has existing exchange, has an app interceptor left prior responses open?" }
```
I assume this is what was intended.

Reply:
Thanks